### PR TITLE
[framework] remove LoggingMiddleware from connection in data fixtures for faster demodata generation

### PR DESCRIPTION
**Pull request by RostislavKreisinger**
| Q             | A
| ------------- | ---
|Description, reason for the PR| remove LoggingMiddleware from connection in data fixtures for faster demo data generation
|New feature| Yes
|[BC breaks](https://docs.shopsys.com/en/latest/contributing/backward-compatibility-promise/)| No
|Fixes issues| ... 
|Have you read and signed our [License Agreement for contributions](https://www.shopsys.com/license-agreement)?| Yes

### CODE REVIEW

File: packages/framework/src/Component/DataFixture/AbstractReferenceFixture.php [hunk 1]
```diff
@@ -13,6 +16,30 @@ abstract class AbstractReferenceFixture implements FixtureInterface
      */
     protected $persistentReferenceFacade;
 
+    /**
+     * @var bool
+     */
+    protected bool $isLoggerTemporarilyDisabled = false;
```
Comment (TomasLudvik) [inline packages/framework/src/Component/DataFixture/AbstractReferenceFixture.php:22]: Why is this necessary? The function is called when the instance is created so this value will be always false, right?

**Review by TomasLudvik — CHANGES_REQUESTED** [review]

File: packages/framework/src/Component/DataFixture/AbstractReferenceFixture.php (hunk 1 above)
Comment (RostislavKreisinger) [reply packages/framework/src/Component/DataFixture/AbstractReferenceFixture.php:22]: you are right, I have consulted with the author and we will remove the review
